File: backend/local-api-module/src/main/java/org/vyatsu/localApiModule/dto/response/api/item/ItemDto.java
package org.vyatsu.localApiModule.dto.response.api.item;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.vyatsu.localApiModule.dto.response.api.UserDto;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ItemDto {
    private Long id;
    private String title;
    private String description;
    private Double firstPrice;
    private UserDto user;
    private String category;
    private String vendorCode;
    private String mpLink;
    private Boolean isDraft;
    private Boolean isActive;
    private LocalDateTime createdAt;
}

File: app/src/main/java/com/bytebusters/android/mpsystem/dto/response/api/item/ItemDto.kt
package com.bytebusters.android.mpsystem.dto.response.api.item

import java.time.LocalDateTime


data class ItemDto (
    val id: Long? = null,
    val title: String? = null,
    val description: String? = null,
    val firstPrice: Double? = null,
    val user: SimpleUserDto? = null,
    val category: String? = null,
    val vendorCode: String? = null,
    val mpLink: String? = null,
    val isDraft: Boolean? = null,
    val isActive: Boolean? = null,
    val createdAt: LocalDateTime? = null
)